step-automation-packages/step-automation-package-controller/src/test/java/step/automation/packages/AutomationPackageManagerOSTest.java

```diff
@@ -108,6 +109,13 @@ public void onDelete(ExecutiontTaskParameters entity) {
         );
     }
 
+    @After
+    public void after() {
+        if (resourceManager != null) {
```

Comment:
@david-stephan 
In default constructor for `LocalResourceManagerImpl` couldn't we mark the root directory to be automatically deleted on JVM exit via `FileUtils#forceDeleteOnExit`? This can solve potential issues if we forget to call explicit  `resourceManager.cleanup()` in some places.

Or it is to dangerous change, what do you think?

Reply:
@iegorov777 LocalResourceMaanger could potentially reuse an exsiting local folder, If think it is the responsability of the client to clean up the folder. However when creating a local resource manager like in the AP staging we could set the flag delete On exist, i.e. in 
ResourceManager resourceManager = new LocalResourceManagerImpl(new File("ap_staging_resources_" + new ObjectId()));
@jeromecomte what do you think?